CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [2.32.0]
+### Changed
+- added lock identity user on creation flag.
+> New optional property added in legal entity stream: `backbase.stream.identity.identity-integration-base-url` to indicate BackBase Identity base URL for clients that integrate with it.
```

Comment:
please make it clear here that this value is mandatory when backbase.stream.legalentity.sink.use-identity-integration=true